Formula/kyma-cli.rb

```diff
@@ -2,9 +2,10 @@ class KymaCli < Formula
   desc "Kyma command-line interface"
   homepage "https://kyma-project.io"
   url "https://github.com/kyma-project/cli/archive/1.24.7.tar.gz"
-  sha256 "c2465d50f0f2c082fc9bc73ccfac4be824cc826980ab263b8b4ce00dcdcc5b9f"
+  sha256 "61dcec274376227c8130b2fd6106c5231f878d949f85972182dadacc0396e460"
   license "Apache-2.0"
-  head "https://github.com/kyma-project/cli.git"
+  revision 1
+  head "https://github.com/kyma-project/cli.git", branch: "main"
```

Comment:
I am curious what does the head and branch signify here.

Reply:
The git repo and branch to use if someone were to run `brew install --HEAD kyma-cli`, which takes the latest commit rather than a released tag.

We don't actively CI test this scenario though, so it's pretty much a YMMV scenario.

Reply:
thanks 👍